columns/Column.js

```diff
@@ -152,6 +156,18 @@ export class Column {
             ...this.agOptions
         };
 
+        if (this.isTreeColumn) {
+            ret.showRowGroup = true;
+            ret.cellRenderer = 'agGroupCellRenderer';
+            ret.cellRendererParams = {
```

Comment:
Exposing the cellRendererParams would be an opportunity to not be too opinionated.

Reply:
As discussed this was changed -- Client can override *all*